main.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"fmt"
+	"github.com/gofiber/fiber/v2"
+	"github.com/gofiber/fiber/v2/middleware/logger"
+	"github.com/kevinpita/go-todo/database"
+	"github.com/kevinpita/go-todo/routes"
+	"log"
+)
+
+func main() {
+	const port = 3000
+
+	app := fiber.New()
+	setupApp(app)
+
+	err := app.Listen(fmt.Sprintf(":%d", port))
```

Comment:
Concatenate it here

Reply:
I would separate the addr, in case you want to modify it

Reply:
Port removed and address constant added [0fd2a94](https://github.com/kevinpita/go-todo/pull/1/commits/0fd2a941b66a67913278408c70bd51f7eab574a4)

App setup already commented before